### DISCUSSION TITLE
How to emulate a tab-panel?

### DISCUSSION THREAD

**Discussion by CrawfishPress** [emilk/egui#1912] [Q&A]
Something like [Kivy Tabbed Panel](https://kivy.org/doc/stable/api-kivy.uix.tabbedpanel.html)? All the tabs have to do, is change a state-flag, say.

I have managed to put up two buttons, in a TopPanel, but any more is beyond me, at this point. Are there any examples of something like this? Thanks.

**Comment by anhosh** [comment]
You might be interested in [egui_dock](https://github.com/Adanos020/egui_dock) I'm working on
